<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConnectAttr</class>
 <widget class="QDialog" name="DialogConnectAttr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>连接器属性编辑</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>4</y>
     <width>261</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>目标</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>25</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Dir1">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/TBD/data/属性窗T节点向下.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="RbConnect1">
      <property name="text">
       <string>1.目标左 2.目标右</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Dir2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/TBD/data/属性窗T节点向上.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="RbConnect2">
      <property name="text">
       <string>1.目标右 2.目标左</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Dir3">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/TBD/data/属性窗T节点向右.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QRadioButton" name="RbConnect3">
      <property name="text">
       <string>1.目标右 2.目标下</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_Dir4">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/TBD/data/属性窗T节点向左.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="RbConnect4">
      <property name="text">
       <string>1.目标左 2.目标下</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="BtnOK">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>232</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>232</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
